crypto/asn1/asn_mstbl.c

```diff
@@ -72,6 +72,8 @@ static int do_tcreate(const char *value, const char *name)
         goto err;
     for (i = 0; i < sk_CONF_VALUE_num(lst); i++) {
         cnf = sk_CONF_VALUE_value(lst, i);
+        if (cnf->value == NULL)
+            continue;
```

Comment:
Should it silently ignore like this, or error? Maybe error is better.

Reply:
Thats a good question, but perhaps beyond the scope of this PR.  All errors from do_tcreate are silently ignored through its call path:
module_run()
 RUN_ONCE(do_load_builtin_modules)
   OPENSSL_load_builtin_modules()
     ASN1_add_stable_module()
      CONF_module_add(..., stbl_module_init)  <= Squashes all return codes, and returns void
        stbl_module_init
         do_tcreate()

Reply:
ping @slontis thoughts above?  I can't see a clear path to exposing a do_tcreate error back to the user

Reply:
modified do_tcreate to raise an error in the event that a value is not parsed